### PR TITLE
Fix readability magic numbers

### PR DESCRIPTION
**Pull request by cbachhuber**
- I guess collecting some parameters in a config file would make sense at some point
- The [cmake check in CI is failing](https://github.com/hofbi/telecarla/pull/10/checks?check_run_id=941580792#step:5:409), even though I didn't touch anything there. Do you know why? My local `cmake-format` runs without reporting any issues

### CODE REVIEW

File: telecarla_gui/source/gui/src/sdl_gui.cpp [hunk 1]
```diff
@@ -73,27 +73,30 @@ void SDL_GUI::renderImage(const SDL_Rect& pos, const sensor_msgs::ImageConstPtr&
...
-                    line.second.data(),
-                    textPos + cv::Point(150, 0),
-                    cv::FONT_HERSHEY_PLAIN,
-                    1.0,
-                    cv::Scalar(125, 125, 125),
-                    1);
-        textPos.y += 20;
+        cv::putText(image, line.first.data(), bottom_left_of_text, cv::FONT_HERSHEY_PLAIN, 1.0, grey, 1);
+
+        cv::Point horizontal_offset(150, 0);  // NOLINT(readability-magic-numbers)
```
Comment (hofbi) [inline telecarla_gui/source/gui/src/sdl_gui.cpp:96]: Why no initializer list here?

File: telecarla_gui/test/gui/gui_parameter_test.cpp [hunk 2]
```diff
@@ -143,7 +143,12 @@ TEST(GuiParameter, getVehicleStatusParameters_WhenValidTestFileWithoutStatus_Emp
 TEST(GuiParameter, getVehicleStatusParameters_WhenValidTestFile_CorrectParameters)
 {
     GuiParameter unit(vehicleStatusTestFilePath);
-    SDL_Rect expectedParameter{0, 720, 640, 200};
```
Comment (hofbi) [inline telecarla_gui/test/gui/gui_parameter_test.cpp:146]: Could you try if this also works `expectedParameter{.x = 0, .y = 0, .w = 0, .h = 0};`

**Comment by hofbi** [comment]
I guess the CI fails because of an update in `cmake-format`. If you run `pip3 install -U cmake_format` and format again the CI should be fixed.

File: telecarla_gui/source/gui/src/sdl_gui.cpp (hunk 1 above)
Comment (cbachhuber) [reply telecarla_gui/source/gui/src/sdl_gui.cpp:96]: Human err, thanks

File: telecarla_gui/test/gui/gui_parameter_test.cpp (hunk 2 above)
Comment (cbachhuber) [reply telecarla_gui/test/gui/gui_parameter_test.cpp:146]: Works! 👍

**Comment by cbachhuber** [comment]
Thanks for the input. All fixed, adapted to camelCase 👍 

Why are you using camelCase for variables? That's quite unusual afaik and doesn't help with distinguishing functions from variables.

**Pull request merged**